### ISSUE TITLE
ctypes.ArgumentError: argument 2: TypeError: expected WINDOWCOMPOSITIONATTRIBDATA instance instead of WindowCompositionAttribute

### ISSUE THREAD

**Issue by littlewhitecloud** [marticliment/win32mica#17]

**Comment by marticliment** [comment]
Can you please post an example of the code that is failing?

Thanks!

**Comment by littlewhitecloud** [comment]
Thanks! Please try this.

`pip install CustomTkinterTitlebar`

And run this code.

```python
from CustomTkinterTitlebar import CTT
from win32mica import ApplyMica

example = CTT()
ApplyMica(example.hwnd, True) # CTT window has hwnd
example.mainloop()
```

**Comment by marticliment** [comment]
This issue will be fixed with version 2.0

**Comment by littlewhitecloud** [comment]
Thanks!
